src/tooltip/styles.css.ts

```diff
@@ -292,14 +292,9 @@ export const tooltipCSS = recipe<ReactAriaRecipe<TooltipRenderProps>>({
 
 export const tooltipFieldButtonCSS = style([
     css({
-        borderRadius: "sm",
-
-        color: "text_low_contrast",
-        flexShrink: "0",
+        display: "flex",
         fontStyle: "bodyMd",
-
-        height: "100%",
-        width: "100%",
+        width: "fit-content",
```

Comment:
My gut says that we might get weird sizing when we keyboard focus this with `fit-content` — might work best if we use explicit height/width to force a square size, e.g. "space_6"?